pycpfcnpj/mask.py

```diff
@@ -0,0 +1,14 @@
+
+from . import cpf
+from . import cnpj
+
+
+def mask_cpf_cnpj(cpf_cnpj: str):
+    cpf_cnpj = "".join([x for x in f'{cpf_cnpj}'])
+
+    if cpf.validate(cpf_cnpj):
+        return f'{cpf_cnpj[:3]}.{cpf_cnpj[3:6]}.{cpf_cnpj[6:9]}-{cpf_cnpj[9:]}'
+    elif cnpj.validate(cpf_cnpj):
+        return f'{cpf_cnpj[:2]}.{cpf_cnpj[2:5]}.{cpf_cnpj[5:8]}/{cpf_cnpj[8:12]}-{cpf_cnpj[12:14]}'
+    else:
```

Comment:
I think if it's not valid, it should raise an exception.

Reply:
I have a case of consultation with the federal revenue base in which the 3 first and 2 last numbers of the cpf are hidden. Oh I leave this mask and not an exception

Reply:
So your code must handle that, not the mask function. The mask function should have only and only one responsibility.